include/nighthawk/adaptive_load/metrics_plugin.h

```diff
@@ -27,6 +35,21 @@ class MetricsPlugin {
    * unavailable.
    */
   virtual absl::StatusOr<double> GetMetricByName(absl::string_view metric_name) PURE;
```

Comment:
We should add a note next to `GetMetricByName` indicating that it is deprecated and why.

Reply:
Done.